tests/k8s/multi-node/tests/deployments/gsg/minikube-gsg-l7-fix.diff

```diff
@@ -0,0 +1,28 @@
+diff --git a/examples/minikube/demo.yaml b/examples/minikube/demo.yaml
+index 0412996..5177c2e 100644
+--- a/examples/minikube/demo.yaml
++++ b/examples/minikube/demo.yaml
+@@ -25,6 +25,8 @@ spec:
+         image: cilium/demo-httpd
+         ports:
+         - containerPort: 80
++      nodeSelector:
```

Comment:
Are you sure we still need this? L7 policy should work across nodes just fine now.